xbmc/video/dialogs/GUIDialogVideoInfo.h

```diff
@@ -107,11 +107,11 @@ class CGUIDialogVideoInfo :
   static bool OnGetFanart(const CFileItemPtr &videoItem);
 
   CFileItemPtr m_movieItem;
-  CFileItemList *m_castList;
-  bool m_bViewReview;
-  bool m_bRefresh;
-  bool m_bRefreshAll;
-  bool m_hasUpdatedThumb;
-  bool m_hasUpdatedUserrating;
-  int m_startUserrating;
+  CFileItemList *m_castList = new CFileItemList;
+  bool m_bViewReview = false;
+  bool m_bRefresh = false;
+  bool m_bRefreshAll = true;
+  bool m_hasUpdatedThumb = false;
+  bool m_hasUpdatedUserrating = false;
+  int m_startUserrating = -1;
```

Comment:
Moreover, like it is now, init is spread over cpp and header which is imo not as good as before where all was in one file. I have to look into two files now to understand whole init setup.